include/z64.h

```diff
@@ -1289,6 +1289,25 @@ typedef struct PreNMIContext {
     /* 0xA8 */ UNK_TYPE unk_A8;
 } PreNMIContext; // size = 0xAC
 
+typedef enum {
+    F_8F = 1,
+    F_7F = 2,
```

Comment:
You can start the enum at 1 (or have an unused entry for 0), but the other ones don't have to be specified.
```suggestion
    F_7F,
```

Reply:
True, I decided to put all the values in just so you can see what number is what floor at a glance. Should I take 'em out?

Reply:
I see, we usually have them in a comment like we do for offsets in structs, but it's mostly a style thing. In this case it doesn't matter since every name has the same size, but in cases like the enums in `z64item.h`, the vertical alignment is better imo. And eventually, for mods, people can more easily clear out the comments if they don't care about maintaining them.

Reply:
Ah cool, I've changed them to comments.